### ISSUE TITLE
Change duration after message displayed

### ISSUE THREAD

**Issue by ektay** [SwiftKickMobile/SwiftMessages#174]
Hi,

I am setting up the message duration to automatic. As per my design if the user clicks on a button in the message view the duration should change to forever. Is it possible to do that?

Thanks
Ekta

**Comment by wtmoose** [comment]
No, not possible. I suggest you use `.forever` and call `hide()` as needed.
